packages/suite/src/components/wallet/FeesAlt.tsx

```diff
@@ -31,13 +33,18 @@ const GreyP = styled(P)`
     color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
 `;
 
-export const Fees = () => {
-    const {
-        account: { symbol },
-        composedLevels,
-        selectedFee,
-    } = useStakeEthFormContext();
-    const transactionInfo = composedLevels?.[selectedFee];
+const HelperTextWrapper = styled(GreyP)`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+interface FeesAltProps {
+    symbol: NetworkSymbol;
+    transactionInfo: PrecomposedTransaction | PrecomposedTransactionCardano | undefined;
+    helperText?: ReactNode;
+}
+
+export const FeesAlt = ({ symbol, transactionInfo, helperText }: FeesAltProps) => {
```

Comment:
Abbreviation for «alternative». There is already a reusable component called Fees. Therefore, this one is supposed to be an alternative version. Renamed to FeesInfo to avoid confusion.

https://github.com/everstake/trezor-suite/pull/6/commits/dd50905ec196c17a0c54ad5b5324e05ba7331362